src/kete_core/src/io/bytes.rs

```diff
@@ -1,8 +1,9 @@
+//! Converting to and from bytes
```

Comment:
This file also got moved, as it is really IO related tools, along with increasing its visibility in the crate.